### PR TITLE
BibFormat: recommendation box new style

### PR DESCRIPTION
**Pull request by ddaze**
- Moves the recommendation box to the bottom.

Signed-off-by: David Zerulla ddaze@outlook.de

### CODE REVIEW

**Comment by ddaze** [comment]
@drjova Can you please have a look at the js, css and also if it works on cds-test with Safari?
Thank you!

File: modules/webstyle/css/invenio.css [hunk 1]
```diff
@@ -4441,26 +4441,36 @@ a.author_orcid_image_link {
...
 /* Record Recommendation */
-.recommendations_content {
-    border-bottom: 1px solid #36c;
-    border-top: 1px solid #36c;
-    margin: 20px 0 0 0;
-    padding: 10px 2%;
+.recommendations {
+    padding: 10px 10px;
+    box-sizing: border-box;
+    border: 1px solid rgb(204, 204, 204);
```
Comment (drjova) [inline modules/webstyle/css/invenio.css]: It might be better to use HEX colors  :)

**Comment by drjova** [comment]
@ddaze on Safari works as supposed to :+1:

**Comment by ludmilamarian** [comment]
IE11 looks ok as well.

File: modules/bibformat/lib/elements/bfe_record_recommendations.py [hunk 2]
```diff
@@ -23,34 +23,31 @@
...
 from invenio.messages import gettext_set_language
 
 
 html_script = """
-<div class="recommendations_content" style="display:none;"></div>
 <script>
 $( document ).ready(function() {
     $.getJSON("%(recommendations_url)s" , function( data ) {
-        var root = $( "div.recommendations_content" );
+        var root = $( "<div/>", {class: 'recommendations'
```
Comment (ludmilamarian) [inline modules/bibformat/lib/elements/bfe_record_recommendations.py]: you are missing a comma after class: 'recommendations', please add it and update the PR

**Comment by ddaze** [comment]
"Show only records with different title" is deployed.

**Comment by ddaze** [comment]
"Show only records with different title" is deployed.

**Comment by ddaze** [comment]
Will be squashed into one commit.
See PR https://github.com/CERNDocumentServer/invenio/pull/169

**Pull request closed without merging**